test/basic.html

```diff
@@ -0,0 +1,65 @@
+<!doctype html>
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE
+The complete set of authors may be found at http://polymer.github.io/AUTHORS
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS
+-->
+<html id="html">
+<head>
+  <meta charset="UTF-8">
+  <title>iron-scroll-target-behavior test</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../web-component-tester/browser.js"></script>
+  
+  <link rel="import" href="x-scrollable.html">
+</head>
+<body>
+
+  <test-fixture id="trivialScrollable">
+    <template>
+      <x-scrollable></x-scrollable>
+    </template>
+  </test-fixture>
+
+  <test-fixture id="trivialScrollableRegion">
+    <template>
+      <div id="scrollingRegion">
+        <x-scrollable id="scrollingRegion"></x-scrollable>
+      </div>
+    </template>
+  </test-fixture>
+
+  <test-fixture id="trivialDocumentScroll">
+    <template>
+      <div id="scrollingRegion">
+        <x-scrollable id="html"></x-scrollable>
+      </div>
+    </template>
+  </test-fixture>
+
+<script>
+
+  suite('basic features', function() {
```

Comment:
yes, sorry that I didn't mention that the test aren't there yet. I will add the test as soon as I can.